README.md

```diff
@@ -1,32 +1,39 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/984a296a-70af-4c2a-9e3f-97924d0a507c/deploy-status)](https://app.netlify.com/sites/cops-hackalog/deploys)
+# COPs Hackalog
 
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+[![Netlify Status](https://api.netlify.com/api/v1/badges/984a296a-70af-4c2a-9e3f-97924d0a507c/deploy-status)](https://app.netlify.com/sites/cops-hackalog/deploys)
+The home for hackathons organized under COPS IIT (BHU), built on Next.js.
 
 ## Getting Started
 
-First, run the development server:
+For setting up the development environment, follow the steps given below.
+
+1. For making changes to the project, fork this repository using the green button on this page.
+
+2. Clone your fork or this repository, as applicable.
```

Comment:
this repository 🤔 for what purpose?

Reply:
Maybe just for trying since this is in general, not just for contributing

Reply:
cool